README.zh-CN.md

```diff
@@ -134,27 +131,31 @@ REPOSITORY         TAG       IMAGE ID       CREATED      SIZE
 apache/eventmesh   latest    f32f9e5e4694   2 days ago   917MB
 ```
 
-#### 2.创建配置文件:
+#### 2.创建配置文件
```

Comment:
`#### 2.创建配置文件` sector can be removed, because it is configuring RocketMQ.

Please remember to add a whitespace between chapter number and heading: 

```suggestion
#### 2. 创建配置文件
```